db/migrate/20200611090846_add_task_lock_index_on_resource_type_and_task_id.rb

```diff
@@ -2,8 +2,8 @@ class AddTaskLockIndexOnResourceTypeAndTaskId < ActiveRecord::Migration[6.0]
   def change
     add_index :foreman_tasks_locks, [:task_id, :resource_type, :resource_id], name: 'index_tasks_locks_on_task_id_resource_type_and_resource_id'
     # These indexes are not needed as they can be gained from partial index lookups
-    remove_index :foreman_tasks_locks, :task_id
-    remove_index :foreman_tasks_locks, :name
-    remove_index :foreman_tasks_locks, :resource_type
+    [:task_id, :name, :resource_type].each do |index|
+      remove_index :foreman_tasks_locks, index unless index_exists?(:foreman_tasks_locks, index)
```

Comment:
shouldn't this be 
```suggestion
      remove_index :foreman_tasks_locks, index if index_exists?(:foreman_tasks_locks, index)
```

Reply:
Yes, the other unless... :man_facepalming: